generators/entity-client/files.js

```diff
@@ -57,7 +57,7 @@ module.exports = {
 function writeFiles() {
     if (this.skipClient) return;
     // write client side files for VueJS
-    this.writeFilesToDisk(vueFiles, this, false, this.fetchFromInstalledJHipster(`../../../../jhipster-vuejs/generators/entity-client/templates/${CLIENT_VUE_TEMPLATES_DIR}`));
+    this.writeFilesToDisk(vueFiles, this, false, 'vue');
```

Comment:
@sahbi-ktifa : I keep the `vue` folder, and revert my last commit about refactoring (+ rebase, so you won't see the history). It will be easier like you did, because if one day we want to merge this module into the main generator-jhipster, I think it will be easier